### PR TITLE
GRIF-193: OSIDB 3.5.0 entities

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR:
* adds support for OSIDB 3.5.0 entities (Flaw CVSS, Flaw Package Versions, Affect CVSS)
* refactors OSIDB entities module into several smaller modules (1 module per 1 entity) for better navigation

### CODE REVIEW

File: griffon/commands/entities/osidb/affects/cvss.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(AffectCVSS))},
+    },
+)
+@click.pass_context
+@progress_bar
+def list_affect_cvss(ctx, affect_id, **params):
+    # TODO: handle pagination
```
Comment (JimFuller-RedHat) [inline griffon/commands/entities/osidb/affects/cvss.py:85]: minor - I believe we handle pagination and output today

File: griffon/commands/entities/osidb/affects/cvss.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    # TODO: handle output
+    session = OSIDBService.create_session()
+
+    params = multivalue_params_to_csv(params)
+    data = session.affects.cvss_scores.retrieve_list(affect_id, **params).results
```
Comment (JimFuller-RedHat) [inline griffon/commands/entities/osidb/affects/cvss.py:90]: random observation - do we think using async is not valid here ?

File: griffon/commands/entities/osidb/affects/cvss.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    help="affect CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "cvss_uuid", help="CVSS UUID.", required=True)
+@request_body_options(endpoint_module=osidb_api_v1_affects_cvss_scores_update, exclude=["uuid"])
+@click.pass_context
+def update_affect_cvss(ctx, affect_id, cvss_uuid, **params):
+    request_body_type = getattr(osidb_api_v1_affects_cvss_scores_update, "REQUEST_BODY_TYPE", None)
+    if request_body_type is None:
+        raise click.ClickException(
```
Comment (JimFuller-RedHat) [inline griffon/commands/entities/osidb/affects/cvss.py:157]: makes me wonder if we ever want to wrap up ClickException in a GriffonException ... just a random muttering

**Review by JimFuller-RedHat — APPROVED** [review]
LGTM (was a bit involved due to the restructure) ... starting to wonder for update scenarios if we want to consider some kind of minimal mocking

File: griffon/commands/entities/osidb/affects/cvss.py (hunk 3 above)
Comment (JakubFrejlach) [reply griffon/commands/entities/osidb/affects/cvss.py:157]: Might be a good idea. Can be done as a side effect of some future PR. Will make a note. Thanks!

File: griffon/commands/entities/osidb/affects/cvss.py (hunk 1 above)
Comment (JakubFrejlach) [reply griffon/commands/entities/osidb/affects/cvss.py:85]: I don't think we handle dynamic output (eg. when due to a include/exclude fields some fields are omitted. We have a issue for that.

File: griffon/commands/entities/osidb/affects/cvss.py (hunk 2 above)
Comment (JakubFrejlach) [reply griffon/commands/entities/osidb/affects/cvss.py:90]: It is valid, but I am still waiting a little bit before we handle the output. I would like to do it together, if that makes sense.

**Comment by JakubFrejlach** [comment]
> LGTM (was a bit involved due to the restructure) ... starting to wonder for update scenarios if we want to consider some kind of minimal mocking

Can you elaborate a little bit please?

File: griffon/commands/entities/osidb/affects/cvss.py (hunk 1 above)
Comment (JimFuller-RedHat) [reply griffon/commands/entities/osidb/affects/cvss.py:85]: no but we do handle output and pagination ... anyhow only nitpicking

**Comment by JimFuller-RedHat** [comment]
re mocking - the decision to not do mocking on reads was based on the fact that we test extensively in the services themselves ... mutation has less coverage and just 'out loud thinking' if we should consider mock tests for update scenarios ... just a random observation unrelated to this PR.

**Pull request merged**